assets/src/edit-story/app/media/media3p/providerConfiguration.js

```diff
@@ -62,8 +63,9 @@ const ProviderType = {
  * @type {Object.<string, ProviderConfiguration>}
  */
 export const PROVIDERS = {
-  [ProviderType.UNSPLASH]: {
-    displayName: 'Unsplash',
+  [ProviderType.IMAGES]: {
+    providerId: 'UNSPLASH',
```

Comment:
Maybe this should be named: providerSource? Will there only be one source per type (i.e. image or video)